app/Notifications/WeeklyStats.php

```diff
@@ -18,7 +19,11 @@ public function via()
 
     public function toSlack()
     {
-        $hits = Hit::forTimePeriod('week');
+        $hits = Hit::forTimePeriod(
+            'week',
+            CarbonImmutable::today(config('app.timezone'))
+                ->setTime(7, 0)
+        );
```

Comment:
Here we're passing in the new second prop, which sets the `end_current_period` value to 7AM UTC. Our weekly cron runs at 8AM UTC.